public/about.html

```diff
@@ -2,17 +2,43 @@
 
 <html lang="en">
 
+<head>
+    <title>RERUM Geolocator</title>
+    <meta charset="UTF-8">
+    <link rel="icon" href="/images/favicon.ico" type="image/ico">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://unpkg.com/chota@latest">
+    <link rel="stylesheet" href="stylesheets/style.css">
+    <script src="components/layout.js"></script>
+</head>
+
+<link rel="stylesheet" href="https://unpkg.com/chota@latest">
+<link rel="stylesheet" href="stylesheets/style.css">
+<header>
```

Comment:
You should not use a `<header>` here.  You will get the `<header>` and `<footer>` the rest of the pages uses automatically when you do `<body is="geo-page">`